### PR TITLE
Add Apprise notification integration

### PR DESCRIPTION
**Pull request by caronc**
**Description:**
Added associated documentation to go with acompanied home-assistant pull request adding Apprise support.

**Pull request in home-assistant:** home-assistant/home-assistant#26868

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/apprise.markdown [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: "Apprise"
+description: "Instructions on how to add Apprise notifications to Home Assistant."
+logo: apprise.png
+ha_category:
+  - Notifications
+ha_release: 0.8.0
```
Comment (klaasnicolaas) [inline source/_components/apprise.markdown]:
```suggestion
ha_release: "0.100"
```

File: source/_components/apprise.markdown [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/apprise.markdown]: Please remove the `redirect_from`

File: source/_includes/custom/grid_item_left.html [hunk 3]
```diff
@@ -41,6 +41,9 @@ <h2 class="title">Observe</h2>
   <a href='/components/arduino/'>
     <img src='/images/supported_brands/arduino.png' alt="Arduino" />
   </a>
+  <a href='/components/notify.apprise/'>
```
Comment (klaasnicolaas) [inline source/_includes/custom/grid_item_left.html]: Please revert this changes

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_includes/custom/grid_item_left.html (hunk 3 above)
Comment (caronc) [reply source/_includes/custom/grid_item_left.html]: Done

File: source/_components/apprise.markdown (hunk 2 above)
Comment (caronc) [reply source/_components/apprise.markdown]: Done

File: source/_components/apprise.markdown (hunk 1 above)
Comment (caronc) [reply source/_components/apprise.markdown]: Done

File: source/_components/apprise.markdown [hunk 4]
```diff
@@ -0,0 +1,80 @@
+---
+title: "Apprise"
+description: "Instructions on how to add Apprise notifications to Home Assistant."
+logo: apprise.png
+ha_category:
+  - Notifications
+ha_release: "0.100"
```
Comment (MartinHjelmare) [inline source/_components/apprise.markdown]:
```suggestion
ha_release: 0.101
```

File: source/_components/apprise.markdown [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+---
+
+The [Apprise service](https://github.com/caronc/apprise/) is a all-in-on solution to open up Home Assistant to _just about_ every Notification platform (such as Amazon SNS, Discord, Telegram, Slack, MSTeams, Twilio etc.
```
Comment (MartinHjelmare) [inline source/_components/apprise.markdown]:
```suggestion
The [Apprise service](https://github.com/caronc/apprise/) is an all-in-on solution to open up Home Assistant to _just about_ every Notification platform (such as Amazon SNS, Discord, Telegram, Slack, MSTeams, Twilio etc.
```

Comment (caronc) [reply source/_components/apprise.markdown]: Done. :slightly_smiling_face:

File: source/_components/apprise.markdown (hunk 4 above)
Comment (caronc) [reply source/_components/apprise.markdown]: Done!

**Review by frenck — APPROVED** [review]
Thanks, @caronc! Awesome addition to the Home Assistant project 😎
